internal/monitors/prometheusexporter/prometheus.go

```diff
@@ -124,7 +127,7 @@ func (m *Monitor) Configure(conf *Config) error {
 	}
 
 	client := &http.Client{
-		Timeout: 10 * time.Second,
+		Timeout: conf.HTTPTimeout * time.Second,
```

Comment:
If you're going to convert it to seconds, then call the config option `httpTimeoutSeconds` and make it an int instead of fudging the meaning of time.Duration.

Reply:
Yeah will just use time.Duration properly like other options, didn't know default groked time units.